index.html

```diff
@@ -0,0 +1,147 @@
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+            <div class="works">
+                <h3 class="works-title">Profesional Art <br>Printing Data</h3>
+                <p class="font-montesori">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tools-container">
+                    <li class="tools bg-btn">html</li>
+                    <li class="tools bg-btn">css</li>
+                    <li class="tools bg-btn">bootstrap</li>
+                    <li class="tools bg-btn">Ruby</li>
+                </ul>
+                
+                <button class="see-project">See Project</button>
+
+            </div>
+           
+
+        </section>
+        <section id="about-me-container">
+
+            <div class="about-me-block">
+                <h3 class="works-title yelow abt-me">About me</h3>
+                <p class="font-montesori  white">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="see-project-btn">Get My Resume</button>
+            </div>
+    
+            <ul  class="categories-container font-montesori white">
+                    <li class="cat-title-1">Language</li>
+                        <ul class="cat-title-2">
+                            <li>Javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    <li class="cat-title-1">Frameworks</li>
+                        <ul class="cat-title-2">
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    <li class="cat-title-1">Skills</li>
+                        <ul class="cat-title-2">
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                </ul>
+                </section>
+                <section id="contact-container">
+                   <h3 class="contact-title font-montesori">I'm always <br>interested in hearing <br>about new projects, <br>so if you'd like to <br>chat please get in touch.</h3>
```

Comment:
- [ ] There will be a milestone in the coming week to add the contact section, kindly remove it and stash it somewhere for now.